CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 
 Unreleased
 ==========
+
+1.0.0:
```

Comment:
```suggestion
1.0.0 (2022-03-30)
==================
```

Needs to be consistent with all of the other packages.

Reply:
Done